gulpfile.js

```diff
@@ -23,6 +23,17 @@ gulp.task('test', ['todo', 'lint'], function performTests() {
     .pipe(mocha({'reporter': 'spec', 'timeout': 100}));
 });
 
+gulp.task('test2', ['lint'], function performTests() {
```

Comment:
Ah ok. Then perhaps a comment explaining why this second test task is here. Other than that looks fine.